### PR TITLE
added user educations consumer

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/EducationrService.Broker/EducationService.Broker.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
```
Comment (VladSpitsin) [inline src/EducationrService.Broker/EducationService.Broker.csproj:6]: fix tabs

**Review by VladSpitsin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

File: test/EducationService.Mappers.UnitTests/Requests/Certificate/PatchDbUserCertificateMapperTests.cs [hunk 2]
```diff
@@ -1,35 +1,33 @@
...
+      var time = DateTime.UtcNow;
+      var userId = Guid.NewGuid();
 
-            _request = new JsonPatchDocument<EditCertificateRequest>(
-                new List<Operation<EditCertificateRequest>> {
+      _request = new JsonPatchDocument<EditCertificateRequest>(
+          new List<Operation<EditCertificateRequest>> {
                     new Operation<EditCertificateRequest>
                     {
                         op = "replace",
```
Comment (irlirla) [inline test/EducationService.Mappers.UnitTests/Requests/Certificate/PatchDbUserCertificateMapperTests.cs:33]: fix tabs?

**Review by irlirla — APPROVED** [review]

File: test/EducationService.Mappers.UnitTests/Requests/Certificate/PatchDbUserCertificateMapperTests.cs (hunk 2 above)
Comment (MariyaSin) [reply test/EducationService.Mappers.UnitTests/Requests/Certificate/PatchDbUserCertificateMapperTests.cs:33]: in VS it is ok

File: src/EducationrService.Broker/EducationService.Broker.csproj (hunk 1 above)
Comment (MariyaSin) [reply src/EducationrService.Broker/EducationService.Broker.csproj:6]: in VS it is ok

**Review by grinakate — APPROVED** [review]

**Pull request merged**